### ISSUE TITLE
运行deepemoji 下的train.py后 准确率很低 上不去，损失函数也没有降低。是什么地方出错了了吗？ 

### ISSUE THREAD

**Issue by ltaatl** [yongzhuo/Keras-TextClassification#52]
Epoch 00017: val_loss did not improve from 2.37730
Epoch 18/20
9999/9999 [==============================] - 929s 93ms/step - loss: 2.4126 - acc: 0.1641 - val_loss: 2.3812 - val_acc: 0.1805

Epoch 00018: val_loss did not improve from 2.37730
Epoch 19/20
9999/9999 [==============================] - 929s 93ms/step - loss: 2.4166 - acc: 0.1628 - val_loss: 2.3788 - val_acc: 0.1805

Epoch 00019: val_loss did not improve from 2.37730
Epoch 20/20
9999/9999 [==============================] - 929s 93ms/step - loss: 2.4126 - acc: 0.1672 - val_loss: 2.3768 - val_acc: 0.1805

**Comment by yongzhuo** [comment]
不知道你配置怎么样的，首先你看看'label'类别数配置得对不对，这个必须配置正确；然后你尝试用自己的word2vec或bert等预训练模式，random随机初始化结果可能会很不稳定。
